package.json

```diff
@@ -75,8 +77,9 @@
     "test": "jest ${1} --forceExit",
     "test-watch": "jest ${1} --watch",
     "lint": "tslint lib/index.ts lib/**/*.ts test/**/*.ts --exclude '**/*.d.ts'",
-    "build": "npm run build:ts && npm run build:components",
+    "build": "rm -rf lib/mediaMappings.* && npm run build:mediaMappings && npm run build:ts && npm run build:components",
     "build:ts": "tsc",
+    "build:mediaMappings": "ts-node scripts/getMediaMappings",
```

Comment:
Since we run `tsc` before this command, I guess we can invoke the js version of this file via node, and therefore don't need `ts-node` as dep?

Reply:
TSC isn't applied to the scripts since they aren't part of the packaged up files

Reply:
But we can add it to our tsconfig, right? Or is there another reason why you would prefer `ts-node`?

Reply:
Well `tsc` is also run after this command because we need to generate the `'./mediaMappings'` file before we can run `tsc` on `RdfParser.ts` wich references it; and we also need to run `tsc` on the generated `'./mediaMappings'`.